npm/vite-dev-server/index-template.html

```diff
@@ -7,43 +7,14 @@
     <title>Components App</title>
   </head>
   <body>
-    <div id="app"></div>
```

Comment:
Why did we change this?

Reply:
This HTML element, `#app`, was never used as far as I can tell. 
Since we needed a `#__cy_root` it felt obvious to me that the id was simply misnamed.